packages/aws-rfdk/lib/deadline/lib/database-connection.ts

```diff
@@ -335,7 +335,7 @@ class DocDBDatabaseConnection extends DatabaseConnection {
     // checking the value of the engineVersion property of that object.
     if (this.props.database.node.defaultChild) {
       const cluster = this.props.database.node.defaultChild! as CfnDBCluster;
-      return cluster.engineVersion?.startsWith('3.6') ?? false;
+      return ( cluster.engineVersion?.startsWith('3.6') || cluster.engineVersion?.startsWith('4.0') || cluster.engineVersion?.startsWith('5.0') )?? false;
```

Comment:
Should this compatibility be tied to Deadline version? Put another way, Deadline 10.3.1.3 [added support for DocumentDB 4/5](https://docs.thinkboxsoftware.com/products/deadline/10.3/1_User%20Manual/manual/release-notes.html#deadline-release); can version 10.3.0 or 10.2.0 use DocumentDB 4/5 successfully?

Reply:
This is a great callout! While deadline versions 10.3.0 & 10.2.0 will support DocumentDB 4/5 since the drivers were updated with the 10.2.0 release https://git.build.awsthinkbox.com/deadline/deadline/-/commit/defe87c803d107793ab8e3d28afa82d2548aa7d0. 10.1.0 will not since the drivers weren't updated then. However, we haven't been updating RFDK with the releases 10.2.0 & 10.3.0 and it's difficult to communicate this partial compatibilities with older versions. So, to make it easier for the customer to comprehend this change we can tie it to the latest version & beyond as per https://docs.thinkboxsoftware.com/products/deadline/10.3/1_User%20Manual/manual/release-notes.html#deadline-release.

Reply:
Looked into the code and discussed this offline with @ddneilson. There isn't a direct way for us to tie this to the deadline version via code since that's not available to us at synthesis time. We should communicate this to the customers by adding a line to the Changelog that the version constraints are updated for the latest version of Deadline 10.3.1 and beyond. @marofke since you're owning this release, this line will need to be added manually to the Changelog in the MCM process, that should address this comment. We should double check with a PM for the wording.